cog.yaml

```diff
@@ -12,7 +12,7 @@ build:
   # a list of packages in the format <package-name>==<version>
   python_packages:
     - "numpy==1.24.2"
-    - "sentencepiece==0.1.97"
+    - "sentencepiece==0.1.99"
```

Comment:
@technillogue, iirc, you did stuff with sentencepiece? Anything to look out for here?

Reply:
no, this is fine